docs/overview.rst

```diff
@@ -195,156 +195,5 @@ It can send a notification to clients using `WebSockets <https://en.wikipedia.or
 or fill a queue of messages in `Redis <http://redis.io/>`_ or execute any custom code of your choice,
 like for sending emails or pinging a third-party.
 
```

Comment:
Personally, I don't think so. The rendered form on Readthedocs has a link to the FAQ on the left side. I think most people look for/go to a FAQ when they have a specific question they want to see answered, which can happen at any point during the document (i.e. not just in the overview, but in concepts, or the tutorial).